ts/features/payments/bizEventsTransaction/saga/handleGetBizEventsTransactions.ts

```diff
@@ -21,7 +21,9 @@ export function* handleGetBizEventsTransactions(
       action,
       {
         size: action.payload.size || DEFAULT_TRANSACTION_LIST_SIZE,
-        "x-continuation-token": action.payload.continuationToken
+        "x-continuation-token": action.payload.continuationToken,
```

Comment:
In this case, the || operator is more appropriate because `size` is a `number` and `0` is a valid value that we want to default to `DEFAULT_TRANSACTION_LIST_SIZE` if encountered. Using `??` would retain `0`, which may lead to unexpected behavior here. Let me know if further clarification would help!